config/locales/en.yml

```diff
@@ -53,6 +53,20 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      ao3_abuse: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work <i>%{title}</i> has been reported as being in violation of the Archive of Our Own's %{tos}."
```

Comment:
Can we reduce the amount of HTML the translators have to work with by using the same `<% pretty_title = "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe %>` approach here that we used in [delete_work_notification](https://github.com/otwcode/otwarchive/blob/master/app/views/user_mailer/delete_work_notification.text.erb)?